core/modules/filter/js/filter.js

```diff
@@ -209,8 +208,9 @@ Backdrop.behaviors.editorImageDialog = {
 
           // Display the library view.
           $('.editor-image-fields').removeClass('editor-image-fields-full');
-          $('.editor-image-library').css({ 'display': 'block' });
-          $('.form-item-image-directory').css({ 'display': 'block' });
+          $('.editor-image-library').load(Backdrop.settings.basePath + 'filter_image_library_ajax');
```

Comment:
I think this might not work with clean URLs disabled. You should perhaps use this instead:

```suggestion
          $('.editor-image-library').load(Backdrop.settings.basePath + '?q=filter_image_library_ajax');
```

I couldn't find a way to convert a menu path to a full URL on the JS side. Sometimes we'll get around this by passing the URL as a setting or as a data attribute. For example you might set a data attribute `data-editor-image-library-url` and then pull it off with `$('.editor-image-library').data('editor-image-library-url')`.